### PR TITLE
Qol 9230 fix gh pages storybook ssi issue

### PR DESCRIPTION
**Pull request by devonpis**
- fix ssi rendering in storybook if there is subfolder in host
- ignore gh-pages branch in circleci workflow
- fix intermittent issue on pngquant module in GH build

### CODE REVIEW

**Review by asifamingov — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -103,7 +103,7 @@
     "html-loader": "^0.5.1",
     "html-webpack-plugin": "^2.28.0",
     "image-webpack-loader": "^3.3.1",
-    "imagemin-pngquant": "^9.0.2",
+    "imagemin-pngquant": "5.0.1",
```
Comment (duttonw) [inline package.json:106]: why did we need to downgrade?

Comment (duttonw) [reply package.json:106]: I see why you needed the package installed. 
https://github.com/imagemin/imagemin-pngquant/issues/46

Does imagemin-pngquant have an alternative that does the same job but less buggy?

**Review by duttonw — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (devonpis) [reply package.json:106]: need to spend time to go through this if we want to replace it.
currently either the testing framework or the dependencies are super fragile in circleci pipeline

better look at moving from circleci to GH actions first

Comment (devonpis) [reply package.json:106]: we are using manual deployment atm so all these stuffs are hidden under a blanket

**Pull request merged**